nise/generators/ocp/ocp_generator.py

```diff
@@ -88,20 +89,51 @@
     "namespace",
     "namespace_labels",
 )
+OCP_ROS_USAGE_COLUMN = (
+    "interval_start",
+    "interval_end",
+    "namespace",
+    "node",
+    "resource_id",
+    "pod",
+    "deployment_name",
+    "image_name",
+    "container_name",
+    "cpu_request_avg_container",
+    "cpu_request_sum_container",
+    "cpu_limit_avg_container",
+    "cpu_limit_sum_container",
+    "cpu_usage_avg_container",
+    "cpu_usage_min_container",
+    "cpu_usage_max_container",
+    "cpu_throttle_avg_container",
```

Comment:
@dgaikwad Please add one more column here for `cpu_throttle_max_container`. It is missing from the Kruize doc I have [commented](https://docs.google.com/document/d/1C7NvTS2Qsj1wY_hfmrlDj9Kud1iJsXfX2EuXnqBFDo4/edit?disco=AAAAohvbiLw) the same in the document.

Reply:
Done